index.js

```diff
@@ -40,11 +40,42 @@ function defaultFormatter(messages) {
   return output.trim();
 }
 
+// load custom rles
+function loadCustomRules(rulesdir) {
+  rulesdir = rulesdir.replace(/\\/g, '/');
+  if (fs.existsSync(rulesdir)) {
+    if (fs.statSync(rulesdir).isDirectory()) {
+      const files = fs.readdirSync(rulesdir);
+      files.forEach(file => {
+        loadRule(path.join(rulesdir, file));
+      });
+    } else {
+      loadRule(rulesdir);
+    }
+  }
+}
+
+// load rule
+function loadRule(filepath) {
+  /* eslint-disable import/no-dynamic-require */  // --> OFF
+  filepath = path.resolve(filepath);
+  try {
+    var module = require(filepath);
```

Comment:
Please change this line to `const module = require(filepath); // eslint-disable-line import/no-dynamic-require`. I also would use a different variable instead of module to prevent shadowing the global module variable. Maybe `rule` instead?